pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2411,18 +2411,17 @@ func handleGlobalConfigModify(ctxArg interface{}, key string,
 		return
 	}
 	log.Infof("handleGlobalConfigModify for %s\n", key)
-	var gcp *types.GlobalConfig
-	debug, gcp = agentlog.HandleGlobalConfig(ctx.subGlobalConfig, agentName,
-		debugOverride)
+	var gcp *types.ConfigItemValueMap
```

Comment:
Why do u need to specifically declare this:

var gcp *types.ConfigItemValueMap

Reply:
I think this was already addressed, but it needs to be specifically declared because debug is variable used over the entire file. So I cannot use := . Therefore, to allow both the global debug variable and this local gcp variable to be set I specifically declare it.

Reply:
I've found that using := for multiple variables is a source of bugs. It will declare new copies of all of the variables, even if some of them already exist. Hence var declarations make sense when multiple variables are involved like in cases like this.